<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculix CAE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>7</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="allColumnsShowFocus">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>29</x>
           <y>29</y>
           <width>91</width>
           <height>141</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="vl">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportINP"/>
    <addaction name="actionExportINP"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionSelectionNodes"/>
    <addaction name="actionSelectionElements"/>
    <addaction name="actionSelectionClear"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelect"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionViewParallel"/>
   <addaction name="actionViewPerspective"/>
   <addaction name="actionViewFront"/>
   <addaction name="actionViewBack"/>
   <addaction name="actionViewTop"/>
   <addaction name="actionViewBottom"/>
   <addaction name="actionViewLeft"/>
   <addaction name="actionViewRight"/>
   <addaction name="actionViewIso"/>
   <addaction name="actionViewFit"/>
   <addaction name="actionViewWireframe"/>
   <addaction name="actionViewSurface"/>
   <addaction name="actionViewSurfaceWithEdges"/>
  </widget>
  <action name="actionImportINP">
   <property name="icon">
    <iconset>
     <normaloff>icons/importINP.png</normaloff>icons/importINP.png</iconset>
   </property>
   <property name="text">
    <string>Import INP file</string>
   </property>
  </action>
  <action name="actionViewFit">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewFit.png</normaloff>icons/viewFit.png</iconset>
   </property>
   <property name="text">
    <string>Fit view</string>
   </property>
  </action>
  <action name="actionSelectionNodes">
   <property name="text">
    <string>Select Nodes</string>
   </property>
   <property name="toolTip">
    <string>Select Nodes</string>
   </property>
  </action>
  <action name="actionSelectionElements">
   <property name="text">
    <string>Select Elements</string>
   </property>
   <property name="toolTip">
    <string>Select Elements</string>
   </property>
  </action>
  <action name="actionSelectionClear">
   <property name="icon">
    <iconset>
     <normaloff>icons/clear_selection.png</normaloff>icons/clear_selection.png</iconset>
   </property>
   <property name="text">
    <string>Clear selection</string>
   </property>
  </action>
  <action name="actionViewParallel">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewParallel.png</normaloff>icons/viewParallel.png</iconset>
   </property>
   <property name="text">
    <string>Parallel View</string>
   </property>
   <property name="toolTip">
    <string>Parallel View</string>
   </property>
  </action>
  <action name="actionViewPerspective">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewPerspective.png</normaloff>icons/viewPerspective.png</iconset>
   </property>
   <property name="text">
    <string>Perspective View</string>
   </property>
   <property name="toolTip">
    <string>Perspective View</string>
   </property>
  </action>
  <action name="actionViewFront">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewFront.png</normaloff>icons/viewFront.png</iconset>
   </property>
   <property name="text">
    <string>Front View</string>
   </property>
   <property name="toolTip">
    <string>Front View</string>
   </property>
  </action>
  <action name="actionViewBack">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewBack.png</normaloff>icons/viewBack.png</iconset>
   </property>
   <property name="text">
    <string>Back View</string>
   </property>
   <property name="toolTip">
    <string>Back View</string>
   </property>
  </action>
  <action name="actionViewTop">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewTop.png</normaloff>icons/viewTop.png</iconset>
   </property>
   <property name="text">
    <string>Top View</string>
   </property>
   <property name="toolTip">
    <string>Top View</string>
   </property>
  </action>
  <action name="actionViewBottom">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewBottom.png</normaloff>icons/viewBottom.png</iconset>
   </property>
   <property name="text">
    <string>Bottom View</string>
   </property>
   <property name="toolTip">
    <string>Bottom View</string>
   </property>
  </action>
  <action name="actionViewLeft">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewLeft.png</normaloff>icons/viewLeft.png</iconset>
   </property>
   <property name="text">
    <string>Left View</string>
   </property>
   <property name="toolTip">
    <string>Left View</string>
   </property>
  </action>
  <action name="actionViewRight">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewRight.png</normaloff>icons/viewRight.png</iconset>
   </property>
   <property name="text">
    <string>Right View</string>
   </property>
   <property name="toolTip">
    <string>Right View</string>
   </property>
  </action>
  <action name="actionViewIso">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewIso.png</normaloff>icons/viewIso.png</iconset>
   </property>
   <property name="text">
    <string>Iso View</string>
   </property>
   <property name="toolTip">
    <string>Iso View</string>
   </property>
  </action>
  <action name="actionViewWireframe">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewWireframe.png</normaloff>icons/viewWireframe.png</iconset>
   </property>
   <property name="text">
    <string>Wireframe View</string>
   </property>
   <property name="toolTip">
    <string>Wireframe View</string>
   </property>
  </action>
  <action name="actionViewSurface">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewSurface.png</normaloff>icons/viewSurface.png</iconset>
   </property>
   <property name="text">
    <string>Surface View</string>
   </property>
   <property name="toolTip">
    <string>Surface View</string>
   </property>
  </action>
  <action name="actionViewSurfaceWithEdges">
   <property name="icon">
    <iconset>
     <normaloff>icons/viewSurfaceWithEdges.png</normaloff>icons/viewSurfaceWithEdges.png</iconset>
   </property>
   <property name="text">
    <string>Surface View With Edges</string>
   </property>
   <property name="toolTip">
    <string>Surface View With Edges</string>
   </property>
  </action>
  <action name="actionExportINP">
   <property name="icon">
    <iconset>
     <normaloff>icons/exportINP.png</normaloff>icons/exportINP.png</iconset>
   </property>
   <property name="text">
    <string>Export INP file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
